test/functorch/test_vmap.py

```diff
@@ -3366,6 +3366,7 @@ def f(x):
             )(x)
             self.assertEqual(output, expected)
 
+    @skipIfTorchDynamo()
```

Comment:
these used to pass, what's going on?

Reply:
The eager backend issue we discussed at HOP meeting, but I think I'll gave up this PR stack since it messed up.